lib/resources/example_json/r4_examples_immunization.rb

```diff
@@ -0,0 +1,109 @@
+               "p&gt;&lt;b&gt;Patient&lt;/b&gt;: Houde, Harry Potter&lt;/p&gt;&lt;p&gt;&lt;b&gt;Status&lt;/b&gt;: completed&lt;"\
+               "/p&gt;&lt;p&gt;&lt;b&gt;Administered by&lt;/b&gt;: Houde, Darrell&lt;/p&gt;&lt;p&gt;&lt;b>Site&lt;/b&gt;: Left "\
+               "Deltoid&lt;/p&gt;&lt;p&gt;&lt;b&gt;Route&lt;/b&gt;: IM&lt;/p&gt;&lt;/div&gt;"
+      },
+      "status": "completed",
+      "vaccineCode": {
+        "coding": [
+          {
+            "system": "http://hl7.org/fhir/sid/cvx",
+            "code": "122",
+            "display": "rotavirus, unspecified formulation",
+            "userSelected": false
+          }
+        ],
+        "text": "rotavirus vaccine"
+      },
+      "patient": {
+        "reference": "Patient/1316020",
+        "display": "Houde, Harry Potter"
+      },
+      "encounter": {
+        "reference": "Encounter/1693915"
+      },
+      "occurrenceDateTime": "2016-09-02T10:13:00-0500",
+      "primarySource": true,
+      "manufacturer": {
+        "reference": "Organization/4297239",
+        "display": "Adams Laboratories"
+      },
+      "lotNumber": "4rft56y",
+      "expirationDate": "2015-02-28",
+      "site": {
+        "coding": [
+          {
+            "system": "http://snomed.info/sct",
+            "code": "37622008",
+            "display": "deltoid region structure (body structure)",
+            "userSelected": false
+          }
+        ],
+        "text": "Left Deltoid"
+      },
+      "route": {
+        "coding": [
+          {
+            "system": "http://snomed.info/sct",
+            "code": "78421000",
+            "display": "Intramuscular route (qualifier value)",
+            "userSelected": false
+          }
+        ],
+        "text": "IM"
+      },
+      "doseQuantity": {
+        "value": 0.5
+      },
+      "performer": [
+        {
+          "function": {
+            "coding": [
+              {
+                "system": "http://terminology.hl7.org/CodeSystem/v2-0443",
+                "code": "AP",
+                "display": "Administering Provider"
+              }
+            ],
+          "text": "Administering Provider"
+          },
+          "actor": {
+            "reference": "Practitioner/1316008",
+            "display": "Houde, Darrell"
+          }
+        }
+      ]
+    }
+
+    R4_IMMUNIZATION_BUNDLE ||= {
+      "resourceType": "Bundle",
+      "id": "9642361d-bab9-444d-a35a-7adce079b8a6",
+      "type": "searchset",
+      "total": 1,
+      "link": [
+        {
+          "relation": "self",
+          "url": "https://fhir-open.devcerner.com/r4/2c400054-42d8-4e74-87b7-80b5bd5fde9f/Immunization?patient=1316020"
+        }
+      ],
+      "entry": [
+        {
+          "fullUrl": "https://fhir-open.devcerner.com/r4/2c400054-42d8-4e74-87b7-80b5bd5fde9f/Immunization/M43434511",
+          "resource": R4_IMMUNIZATION_ENTRY
+        }
+      ]
+    }
+  end
+end
```

Comment:
add a newline to this and lib/resources/r4/immunization.yaml

Reply:
Added. https://github.com/cerner/fhir.cerner.com/pull/255/commits/7f7bd5fd0fd7d7d18677ca1d5618107ccb707330#diff-b4a3f148f7c74d28ca272418093bb464R109